cms/templates/js/content-group-details.underscore

```diff
@@ -1,13 +1,58 @@
-<div class="collection-details">
+<div class="collection-details wrapper-group-configuration">
     <header class="collection-header">
         <h3 class="title">
-            <%- name %>
+            <a href="#" class="toggle group-toggle <% if (showContentGroupUsages){ print('hide'); } else { print('show'); } %>-groups">
+                <i class="ui-toggle-expansion icon fa fa-caret-<% if (showContentGroupUsages){ print('down'); } else { print('right'); } %>"></i>
+                <%= name %>
+            </a>
         </h3>
     </header>
 
-    <ul class="actions">
+   <ol class="collection-info group-configuration-info group-configuration-info-<% if(showContentGroupUsages){ print('block'); } else { print('inline'); } %>">
+     <% if (!_.isUndefined(id)) { %>
+         <li class="group-configuration-id"
+             ><span class="group-configuration-label"><%= gettext('ID') %>: </span
+             ><span class="group-configuration-value"><%= id %></span
+         ></li>
+     <% } %>
+     <% if (!showContentGroupUsages) { %>
+          <li class="group-configuration-usage-count">
+             <%= usageCountMessage %>
+         </li>
+     <% } %>
+   </ol>
+
+    <ul class="actions group-configuration-actions">
         <li class="action action-edit">
             <button class="edit"><i class="icon fa fa-pencil"></i> <%= gettext("Edit") %></button>
         </li>
+        <% if (_.isEmpty(usage)) { %>
+            <li class="action action-delete wrapper-delete-button">
+                <button class="delete action-icon"><i class="icon fa fa-trash-o"></i><span><%= gettext("Delete") %></span></button>
+            </li>
+        <% } else { %>
+            <li class="action action-delete wrapper-delete-button" data-tooltip="<%= gettext('Cannot delete when in use by an unit') %>">
```

Comment:
@catong We are using the same label and button in experiment groups including the detailed hover text.